teamA/kimTaeksoo/app.js

```diff
@@ -22,13 +22,12 @@ app.get("/ping", (req, res, next) => {
   res.json({ message: "success" });
 });
 
-myDataSource.initialize().then(() => {
-  console.log("data source has been init");
-});
-
 const start = () => {
   server.listen(PORT, () => {
     console.log(`Open Server with ${PORT}`);
+    myDataSource.initialize().then(() => {
+      console.log("data source has been init");
+    });
```

Comment:
error 발생 시 코드도 같이 작성해보면 좋을 것 같네요!

```suggestion
    await myDataSource.initialize()
      .then(() => {
        console.log("data source has been init");
      }.catch(err) => {
        console.log("data source failed to initialize", err);
        myDataSource.destroy();  
      });
```